### ISSUE TITLE
clang.cindex.LibclangError: libclang.so

### ISSUE THREAD

**Issue by robertjwright** [elliotchance/c2go#4]
I get the following error after installing llvm, clang, libclang-dev via package manager and installing the python bindings from the link in the readme (tar file, then python setup.py install):

clang.cindex.LibclangError: libclang.so: cannot open shared object file: No such file or directory. To provide a path to libclang use Config.set_library_path() or Config.set_library_file().

Which distro did you use?
Which packages did you install?
How did you install the python bindings?

**Comment by elliotchance** [comment]
So... here comes the embarrassing part. I don't actually know what I did to set it up correctly, at least I couldn't replicate my steps for the travis build (mac => linux).

However, not all is lost. I am almost finished working on a refactor that removes the need for any binary bindings or python libraries. It uses the natural `clang` output from the command line to build the AST. This should make it a lot less brittle (clang AST API is not stable and has changed many times) and not require any libraries in the middle.

Stay tuned, I will post a PR for those changes shortly. I won't close this issue until that PR has been merged in 😄
